static/sass/_network_map.scss

```diff
@@ -0,0 +1,41 @@
+#network-map {
```

Comment:
Could we assign a class and style by it? It's not a good practice to style by ID.

Reply:
I needed the id in JS, but we can have and ID and a class.